projects/plugins/jetpack/modules/likes/jetpack-likes-settings.php

```diff
@@ -132,21 +134,25 @@ public function meta_box_save( $post_id ) {
 		}
 
 		if ( 'post' == $_POST['post_type'] ) {
-			if ( !current_user_can( 'edit_post', $post_id ) ) {
+			if ( ! current_user_can( 'edit_post', $post_id ) ) {
 				return $post_id;
 			}
 		}
 
 		// Record a change in like status for this post - only if it contradicts the
 		// site like setting. If it doesn't contradict, then we delete the new individual status.
+		// phpcs:ignore WordPress.Security.NonceVerification.Missing -- we are only checking if $_POST['wpl_enable_post_likes'] is not empty, so ignoring the nonce verification. 
```

Comment:
But we're checking it to determine whether to call `update_post_meta()`. I think this needs better justification than is provided in this comment.

Same for the next few below.

Reply:
In general, implementing nonce checks would probably be the best thing to do. https://developer.wordpress.org/themes/theme-security/using-nonces/ seems to have a decent overview.

But what's the context of this code? This method seems to run from the "save_post" and "edit_attachment" actions. Can we assume that by the time WordPress fires one of those actions, it already did a nonce check itself? If so, then that would be sufficient justification for ignoring the sniff.